### PR TITLE
Members: add responsive layout to tablet resolution [DEPENDS ON https://github.com/inveniosoftware/invenio-communities/pull/671]

### PR DESCRIPTION
**Pull request by Pineirin**
Fix mobile UI issues

closes: [inveniosoftware/invenio-app-rdm#661](https://github.com/inveniosoftware/invenio-communities/issues/661)
closes: https://github.com/inveniosoftware/invenio-communities/issues/646
depends on: https://github.com/inveniosoftware/invenio-communities/pull/671

Screenshot from 2022-05-18 15-49-10
Screenshot from 2022-05-18 15-49-21
Screenshot from 2022-05-18 15-49-34

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides [hunk 1]
```diff
@@ -77,3 +77,17 @@
     }
   }
 }
+
+.mobile-table-cell .ui.dropdown{
+  @media all and (max-width: @largestMobileScreen) {
+    min-width: 0em !important;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:83]: please change to `7em` otherwise it is a bit strange

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides [hunk 2]
```diff
@@ -77,3 +77,17 @@
...
+
+    & > .menu {
+      min-width: 14em !important;
+      left: auto !important;
+      right: 0px !important;
+      border-top-width: 1px !important;
+      border-radius: 0.28571429rem 0rem 0.28571429rem 0.28571429rem;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:90]: please dont hardcode values. There is a variable specifying border-radius. How to find the variable: https://inveniordm.docs.cern.ch/customize/styling-theme/

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides [hunk 3]
```diff
@@ -77,3 +77,17 @@
     }
   }
 }
+
+.mobile-table-cell .ui.dropdown{
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:81]: this change should be in dropdown.overrides

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides [hunk 4]
```diff
@@ -77,3 +77,17 @@
...
+  @media all and (max-width: @largestMobileScreen) {
+    min-width: 0em !important;
+
+    & > .menu {
+      min-width: 14em !important;
+      left: auto !important;
+      right: 0px !important;
+      border-top-width: 1px !important;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:89]: why are these needed? could you avoid using `important` ?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides (hunk 3 above)
Comment (Pineirin) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:81]: It is on it.

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides (hunk 4 above)
Comment (Pineirin) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:89]: The rule I'm overriding was marked as important, so I don't think I can. This rule is needed because in semantic ui react, as the dropdown should have the same width than the dropped text, not top margin is required.

Comment (Pineirin) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:89]: The problem with this approach is that there is also a blue line on the bottom of the dropdown when you expand (see screenshot 3). I'm not a fan of this, as it is not consistent with the other views, but after a while trying things I could only find one alternative solution (using before) and it feels quite hackish.

Comment (kpsherva) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides:89]: where was it marked as important ?

**Review by kpsherva — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]
